### PR TITLE
Issue/1010 unlimited

### PR DESCRIPTION
**Pull request by kardan**
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: backend/src/akvo/lumen/lib/tier_impl.clj [hunk 1]
```diff
@@ -3,7 +3,9 @@
    [ring.util.response :refer [response]]))
 
 (defn all [tenant-conn]
-  (response {"tiers" {"standard" {"numberOfExternalDatasets" 5
+  (response {"tiers" {"unlimited" {"numberOfExternalDatasets" "*"
```
Comment (jonase) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: Should we use `nil` instead of `"*"` for "no limit"?

Comment (kardan) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: I was not sure what would be best but thought  * made more sense from the UI point of view. Rendering "nil" (not sure that's what you suggest) makes me think something is broken.

Comment (kardan) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: Maybe we should say "unlimited"

Comment (kardan) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: @nadiagorchakova thoughts?

Comment (jonase) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: What we render is different from what we the data structure contains. I was just thinking that it's a bit strange to have key/value pairs where the the value is sometimes a number and sometimes a string. `null` to me would mean "no value" and would therefor be unlimited.

We can still render *, unlimited, ∞ or whatever

Comment (kardan) [inline backend/src/akvo/lumen/lib/tier_impl.clj]: Agree!

**Comment by jonase** [comment]
`RELEASE NOTES`: Added "unlimited" tenant plan

**Pull request merged**